extra/html-extra/HtmlExtension.php

```diff
@@ -34,7 +36,7 @@ public function getFilters(): array
     public function getFunctions(): array
     {
         return [
-            new TwigFunction('html_classes', 'twig_html_classes'),
+            new TwigFunction('html_classes', [static::class, 'htmlClasses']),
```

Comment:
There is no need for the late static binding, the class is final.
```suggestion
            new TwigFunction('html_classes', [self::class, 'htmlClasses']),
```